tox.ini

```diff
@@ -0,0 +1,80 @@
+[tox]
+requires =
+    tox>=4
+env_list = coverage, build, firmware, docs, format, type, py{39,310,311,312}
+
+[testenv]
+description = run unit tests
+deps = 
+    -r{toxinidir}/requirements.txt
+    pytest==7.0.1
```

Comment:
I think we should create a requirements file for these dependencies.

Reply:
To avoid defining dependencies in two places (tox.ini and requirements.txt). If we define the testing and dev dependencies in a txt file, we could simply use them here.

Reply:
Well, I wanted to remove the requirements.txt as well. I kept it because sometimes we want to point to a specific branch of ingenialink as it is now in the develop branch.

Reply:
requirements.txt has no sense anymore. For install ingenialink develop branch we can add a file to these cases. But usually it will not be necessary

Reply:
Ok, I added a custom_requirements file to keep all requirements in the tox.ini file.

Reply:
@WoodElv found a way to avoid using a custom_requirements file: [69d99a4](https://github.com/ingeniamc/ingeniamotion/pull/145/commits/69d99a49db47db6a0f88eba31de8e21befd6c815)